docs/static/llama-stack-spec.yaml

```diff
@@ -12170,6 +12171,29 @@ components:
       - items
       title: ConversationItemCreateRequest
       type: object
+    ProcessFileResponse:
+      description: |-
+        Response model for file processing operation.
+
+        Returns a list of chunks ready for storage in vector databases.
+        Each chunk contains the content and metadata.
+      properties:
+        chunks:
+          description: Processed chunks from the file. Always returns at least one chunk.
+          items:
+            $ref: '#/components/schemas/Chunk'
```

Comment:
@franciscojavierarceo what happens here when `Chunk.embedding` becomes required?

Reply:
I'm going to reconcile that. Chunk should not require embeddings in File Processor but it should in VectorIO.